src/AnnotationDialog.js

```diff
@@ -507,16 +500,8 @@ class AnnotationDialog extends EventEmitter {
      * @return {void}
      */
     clickHandler(event) {
```

Comment:
Should this also be passed to `Annotation`?

Reply:
This handles all the click events in the dialog, I removed anything related to the annotation itself (which was mostly related to deleting annotations from a thread)